packages/apps/demo/src/Legend.tsx

```diff
@@ -103,21 +103,20 @@ export const Legend = (props: LegendProps) => {
   const [activeTab, setActiveTab] = useState<number>(0);
   return (
     <>
-      <Tooltip title={'Map Options'}>
```

Comment:
Unrelated change:

This removes the `<Tooltip>` wrapper and adds a `title` attribute to the IconButton so that the Map Options button looks/feels like the other MapLibre buttons.